resources/metadata/abapEnvironmentCreateSystem.yaml

```diff
@@ -179,6 +179,46 @@ spec:
           - STAGES
           - STEPS
           - GENERAL
+      - name: abapSystemParentServiceLabel
```

Comment:
Same here: ask Welf if should be provided, no external use case (?)
CLD is internal